src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,90 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size = 0;
+    private Node<K, V>[] table;
```

Comment:
The diamond operator is missing the type parameters here. It should be initialized as `new Node<K,V>[DEFAULT_CAPACITY]` to maintain type safety.